test/categorical_algebra/Diagrams.jl

```diff
@@ -71,6 +71,18 @@ d = dom(f)
 @test codom(op(f)) == op(dom(f))
 @test op(g)⋅op(f) == op(f⋅g)
 
+# param_compose, should probably be in FinCats
+####################
+C = FinCat(parallel_arrows(Graph,2))
+D = FinCat(FinSet(2))
+F = FinDomFunctor([FinSet(2),TypeSet(Float64)],[FinDomFunction([1.0,2.3]),FinDomFunction([-π,π])],C)
+G = FinFunctor([1,1],D,C)
+H = FinFunctor([2,2],D,C)
+α = CategoricalAlgebra.FinCats.FinTransformationMap([1,FreeSchema.Attr{:nothing}([],[])],G,H)
```

Comment:
Should create an issue to track how we can refactor this design in the future. One possibility is allowing functors and transformations to be partial by default (like we do for functions in acsets).

Reply:
Welp

Reply:
Probably should've left this for later, but it's resolved!